opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -0,0 +1,172 @@
+        self,
+        instrument_type: Optional[Type[Instrument]] = None,
+        instrument_name: Optional[str] = None,
+        meter_name: Optional[str] = None,
+        meter_version: Optional[str] = None,
+        meter_schema_url: Optional[str] = None,
+        name: Optional[str] = None,
+        description: Optional[str] = None,
+        attribute_keys: Set[str] = None,
+        aggregation: Optional[Type[Aggregation]] = None,
+    ):
+        """
+        An instance of `View` is an object that can perform two actions:
+
+        1. Match instruments: When an instrument matches a view, measurements
+           received by that instrument will be processed.
+        2. Customize metric streams: A metric stream is identified by a match
+           between a view and an instrument and a set of attributes. The metric
+           stream can be customized by certain attributes of the corresponding
+           view.
+
+        The attributes documented next serve one of the previous two purposes.
+
+        Args:
+            instrument_type: This is an instrument matching attribute: the class
+            the instrument must be to match the view.
+            instrument_name: This is an instrument matching attribute: the name
+            the instrument must have to match the view.
+            meter_name: This is an instrument matching attribute: the name
+            the instrument meter must have to match the view.
+            meter_version : This is an instrument matching attribute: the
+            version the instrument meter must have to match the view.
+            meter_schema URL : This is an instrument matching attribute: the
+            schema URL the instrument meter must have to match the view.
+            name: This is a metric stream customizing attribute: the name of
+            the metric stream. If `None`, the name of the instrument will be
+            used.
+            description: This is a metric stream customizing attribute: the
+            description of the metric stream. If `None`, the description of the
+            instrument will be used.
+            attribute_keys: This is a metric stream customizing attribute: this
+            is a set of attribute keys. If not `None` then only the measurement
+            attributes that are in `attribute_keys` will be used to identify
+            the metric stream.
+            aggregation: This is a metric stream customizing attribute: every
+            metric stream has an aggregation instance, this is the class of
+            aggregation this instance will be. If `None` the default
+            aggregation class of the instrument will be used.
+        """
+
+        if (
+            instrument_type
+            is instrument_name
+            is meter_name
+            is meter_version
+            is meter_schema_url
+            is None
+        ):
+            raise Exception(
+                "Some instrument selection criteria must be provided"
```

Comment:
```suggestion
                "Some instrument selection criteria must be provided for View {name}", 
```

Reply:
But that can lead to different error messages, right?

Reply:
Is that not a problem? Just need a way to help the user troubleshoot.

Reply:
Ok, refactored the message :+1: